src/lib.rs

```diff
@@ -497,6 +498,9 @@ where
     /// be worth experimenting to see which of the two methods is faster.
     ///
     /// [`most_common_ordered`]: Counter::most_common_ordered
+    ///
+    /// # Panics
+    /// Preconditions ensure that the current implementation does not panic. 
```

Comment:
Is the clippy logic looking for the existence of a "Panics" section in the docs if there is a `.expect` in the function? I'd prefer to simply suppress that lint. Adding a big, bold-face documentation item "**Panics**: the current implementation does not panic" is unhelpful and pointless.

Reply:
I suppressed the lint with a comment that reflects how it doesnt panic currently.

Reply:
That suppresses the lint right now, but it introduces distracting and potentially confusing documentation. The default expectation of a function is that it does not panic, unless it has a "Panics" section in the documentation. Adding a "Panics" section which just says "this does not panic" does not help someone reading the documentation, but it has the potential to confuse them. Better to omit that section of the documentation and instead suppress the lint with an `#[allow(clippy::wants_pointless_documentation)]` (not the actual lint name) annotation.